index.html

```diff
@@ -7446,6 +7446,14 @@ <h5>Presentational Role Inheritance</h5>
 			<rdef>searchbox</rdef>
 			<div class="role-description">
 				<p>A type of textbox intended for specifying search criteria. See related <rref>textbox</rref> and <rref>search</rref>.</p>
+				<p>
+					User agents MUST expose the value of elements with role <code>searchbox</code> to <a>assistive technologies</a>. The value of a <code>searchbox</code> is represented by one of the following:
+				</p>
+					<ul>
+						<li>If the <code>searchbox</code> element is a host language element that provides a value, such as an HTML <code>input</code> element, the value of the searchbox is the value of that element.</li>
+						<li>Otherwise, the value of the <code>searchbox</code> is represented by its descendant elements and can be determined using the same method used to compute the name of a <rref>button</rref> from its descendant content.</li>
```

Comment:
is there something special about button in particular that it is uniquely called out, instead of alluding to the textbox/searchbox needs to calculate its value based on its subtree, similar to roles who derive their name from content?

Reply:
This is also a copy of the [combobox](https://w3c.github.io/aria/#combobox) text. If we change it here, we should change it there. The link to the button role at the combobox text is also slightly misleading, because it suggests that the link points to an explanation of how the button's name is derived from its content, but this is not the case (for that, the AccName would have to be linked)

Reply:
yeh.  tbh i should have paid more attention to that and even though this text is coming from combobox, I would then say that my questions also relate to combobox then.

Reply:
@scottaohara thanks, I'll close this PR and get these changes to the same content in one PR.  @scottaohara @JAWS-test what do you think of the following verbiage?
						<li>Otherwise, the value of the <code>searchbox</code> is determined by the <a href="https://www.w3.org/TR/accname-1.1/" >Accessible Name and Description Computation</a>.</li>

Reply:
@chlane I am not sure if it is useful to refer to the AccName as it is too complicated to understand and in my opinion wrong:
* Imagine that in the textbox there is a div. A div is not allowed to have a label, with that the textbox then has no value?
* Imagine that there is an element in the textbox, which has text content and is labeled by aria-label at the same time. Should the value be the text (like in Edge and Chrome) or the value in the aria-label (like in Firefox)?

Until the AccName resolves such inconsistencies, maybe we should just say the text content is the value, right?

Reply:
@JAWS-test thanks, the verbiage "using the same method used to compute the 'name' of a button" has led me to conflate name and value. Do you think "Otherwise, the value is determined by its text content" will work?

Reply:
@chlane After all, it's not about the wording, it's about the method used. Simply taking the text content would be different than taking the name of a button as the value. I was just expressing my concern about what happens when we use the button method. But I'm not in a position to decide which method is the right one. That should be decided by the ARIA WG.

Reply:
TODO: At the weekly meeting we decided to 1) create a new issue in accname to explain how values are grabbed and 2) remove the current verbiage that describes getting the value for comboboxes from combobox, searchbox and textbox